src/sentry/api/endpoints/organization_force_auto_assignment.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import ratelimits as ratelimiter
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import Organization
+from sentry.models.group import Group
+from sentry.models.projectownership import ProjectOwnership
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+
+
+@region_silo_endpoint
+class OrganizationForceAutoAssignmentEndpoint(OrganizationEndpoint):
+
+    rate_limits = {"PUT": {RateLimitCategory.ORGANIZATION: RateLimit(1, 60)}}  # 1 rpm
+
+    def put(self, request: Request, organization: Organization) -> Response:
+        """
+        Endpoint for forcing autoassignment to run for specified group ids.
+        This is for if a user incorrectly manually assigns a group and wants autoassignment to run.
```

Comment:
This seems like a parameter addition to an existing api instead of a new one. Can you point me to the assign api?

Reply:
There is a manual assign API [here](https://github.com/getsentry/sentry/blob/1cb4428d34d556792e166d0119eca7f41cbe42e7/src/sentry/api/endpoints/group_details.py#L277), but this API is different as we want to call auto-assignment. Auto-assignment is usually called in a task [here](https://github.com/getsentry/sentry/blob/1cb4428d34d556792e166d0119eca7f41cbe42e7/src/sentry/tasks/post_process.py#L981) and not in an API

Reply:
> Auto-assignment is usually called in a task [here](https://github.com/getsentry/sentry/blob/1cb4428d34d556792e166d0119eca7f41cbe42e7/src/sentry/tasks/post_process.py#L981) and not in an API

Thanks for the pointers. You're calling it in an api now so it seems like a good fit for the API. For example can assignedTo param get user:AUTO_ASSIGN or even add a new enum that assignmentMethod